packages/app-admin-ui/client/pages/Home/index.js

```diff
@@ -56,6 +24,17 @@ const HomepageListProvider = () => {
   const query = gqlCountQueries(lists);
   const { data, error } = useQuery(query, { fetchPolicy: 'cache-and-network', errorPolicy: 'all' });
 
+  const [cellWidth, setCellWidth] = useState(3);
+  const { ref: measureElement } = useResizeObserver({
+    onResize: ({ width }) => {
+      // requestAnimationFrame works around a weird 'ResizeObserver loop limit exceeded' error.
```

Comment:
`use-resize-observer` has a throttle\debounce method for this: https://github.com/ZeeCoder/use-resize-observer#throttle--debounce

I think we want to throttle it to about 15 fps ie around 60ms

Reply:
Switched to using `lodash.throttle` as the docs suggest.

Reply:
Looks like `requestAnimationFrame` was still required after all.

Reply:
Ok, `throttle` doesn't fix the problem (see https://github.com/WICG/ResizeObserver/issues/38). `requestAnimationFrame` was needed after all.